packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -6201,6 +6201,10 @@ export interface IScrollablePaneContext {
         sortSticky: (sticky: Sticky, sortAgain?: boolean) => void;
         notifySubscribers: (sort?: boolean) => void;
         syncScrollSticky: (sticky: Sticky) => void;
+        usePlaceholderForSticky: (isTop: boolean) => boolean;

```

Comment:
You may consider using a string union enum if the boolean parameter is too opaque.

```ts
type PlaceholderPosition = 'top' | 'bottom';
...
usePlaceholderForSticky(position: PlaceholderPosition) => boolean;
```

I only suggest because of future flexibility and the number of comments present throughout saying whether or not `isTop`.

Reply:
> You may consider using a string union enum if the boolean parameter is too opaque.
> type PlaceholderPosition = 'top' | 'bottom';
> ...
> usePlaceholderForSticky(position: PlaceholderPosition) => boolean;
> I only suggest because of future flexibility and the number of comments present throughout saying whether or not isTop.

sure